tests/rabbitmq/test_basic.py

```diff
@@ -0,0 +1,19 @@
+def test_rabbitmq_basic(rabbitmq):
+    exchange = "testsuite_exchange"
+    queue = "testsuite_queue"
+    routing_key = "testsuite_routing_key"
+
+    with rabbitmq.get_channel() as channel:
```

Comment:
is there asyncio driver?

Reply:
There is, https://github.com/mosquito/aio-pika

Should i go with it or leave it be?

Reply:
I'd prefer to use aio driver if it's stable and supported

Reply:
Ok, i'll change the code accordingly